### PR TITLE
Senate cartogram

### PR DESCRIPTION
**Pull request by njdehoog**
* Add support for rendering inside strokes on polygon features
* Add `anchor` property to text style, and draw text features accordingly
* Add story for senate cartogram

### CODE REVIEW

File: src/lib/components/molecules/canvas-map/lib/styles/Text.js [hunk 1]
```diff
@@ -1,6 +1,60 @@
...
+  TOP: "top",
+  BOTTOM: "bottom",
+  LEFT: "left",
+  RIGHT: "right",
+  CENTER: "center",
+  TOP_LEFT: "top-left",
+  TOP_RIGHT: "top-right",
+  BOTTOM_LEFT: "bottom-left",
+  BOTTOM_RIGHT: "bottom-right",
+}
```
Comment (egargan) [inline src/lib/components/molecules/canvas-map/lib/styles/Text.js:28]: Nitpick: I tend to prefer string unions over enums like this: we don't have to import anything to provide a value, and it's easier to troubleshoot runtime issues (e.g. when we see a stray `top-left`, we don't have to know that it came from `TextAnchor.TOP_LEFT`, it'll appear as `top-left` in our source code)

**Review by egargan — APPROVED** [review]
LGTM, great work!

File: src/lib/components/molecules/canvas-map/lib/styles/Text.js (hunk 1 above)
Comment (njdehoog) [reply src/lib/components/molecules/canvas-map/lib/styles/Text.js:28]: Forgot to read this before I hit merge, sorry. Could you give me an example of what that would look like? I'm not sure I understand.

Comment (egargan) [reply src/lib/components/molecules/canvas-map/lib/styles/Text.js:28]: No worries! Not a blocking issue at all.

It'd look something like this...

```js
/** @typedef { "top" | "bottom" | "left" | "right" | "center" | "top-left" | ... } TextAnchor */
```

Then further down, you'd just use this type as the parameter type...

```js
export class Text {
  /**
   * Create a text element style
   * @constructor
   * @param {TextStyle} options - Style options
   */
  constructor(options) {
    this.content = options?.content
    this.anchor = options?.anchor || "center"
    // ...
  }
}
```

When dealing with these `TextAnchor` values, you'll still get completion, and Typescript will still shout at you if you provide the wrong value.

Comment (njdehoog) [reply src/lib/components/molecules/canvas-map/lib/styles/Text.js:28]: Nice one, thanks! That's a much cleaner solution. I'll use it from now on.

**Pull request merged**